### ISSUE TITLE
pausemenu music none option acts like a music file

### ISSUE THREAD

**Issue by lazyremixmanordummyplayinggmod** [ShadowMario/FNF-PsychEngine#13844]
### Describe your bug here.

Mind you, if you are modding, please try testing your bug in a clean version of the engine instead, this ensures the bug is actually caused by the engine itself.

crashes bc none acts like a music file for some reason
![Screenshot 2023-12-24 6 51 15 PM](https://github.com/ShadowMario/FNF-PsychEngine/assets/97311877/1b2b9569-df6c-426c-83f5-4d66b0337625)

### Command Prompt/Terminal logs (if existing)

```bash
Uncaught Error: Error loading sound from file: ./assets/shared/music/none.ogg
Please report this error to the GitHub page: https://github.com/ShadowMario/FNF-PsychEngine
```

### Are you modding a build from source or with Lua?

Not modding

### Which branch of the engine are you using?

Latest (main) 0.7.2

### What is your build target?

Windows

### Did you edit anything in this build? If so, mention or summarize your changes.

none

**Comment by lazyremixmanordummyplayinggmod** [comment]
i tryed changing then changing bac and this did not work

**Comment by ShadowMario** [comment]
Should be fixed now, redownload 0.7.2h
